sdk/servicebus/azservicebus/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/Azure/azure-service-bus-go
```

Comment:
This will need to be updated.

Reply:
So this would be:

```
github.com/Azure/azure-sdk-for-go/sdk/service/servicebus/azservicebus
```

Is that the correct style for our modules? (oof, that's a long path)

Reply:
I believe `github.com/Azure/azure-sdk-for-go/sdk/servicebus/azservicebus` or `github.com/Azure/azure-sdk-for-go/sdk/azservicebus` could work too. I don't think you need the `/service/` portion.